packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -318,6 +319,11 @@ class MenuAnchor extends StatefulWidget {
   /// to rebuild this child when those change.
   final Widget? child;
 
+  /// The padding between the edge of the safe area and the menu panel.
+  ///
+  /// Defaults to EdgeInsets.all(8).
+  final EdgeInsetsGeometry? viewPadding;
```

Comment:
nit: I know viewPadding has been used in _Submenu, but for a public API, I just feel this name is a bit confusing. It sounds like this is the same as MenuStyle.padding. How about reservedPadding? Please let me know if there are more name proposals!

Reply:
`reservedPadding` seems great to me!
I will push a commit asap.